### ISSUE TITLE
Add group enrolment feature

### ISSUE THREAD

**Issue by DownesAyr** [ndunand/moodle-enrol_attributes#15]
Hi there,

It would be nice to see an ability to add users into a particular group on successful profile field match.

We have the same courses run across different campuses you see and it is useful to filter the students by group, so it would be very handy for us to have this process automated by your plugin.

Cheers

**Comment by ndunand** [comment]
Hi,

Thanks, I'll keep that in mind.

If you have any ressources in terms of manpower this could speed things up. I work for my employer and we make this available to everyone but we're probably not (in the foreseeable future) going to work very fast on something our user base hasn't requested.

**Comment by gemguardian** [comment]
Hi, just wondering if you could give a cost indication since I might have a client that would be interested in this option. 

Thank you
Gemma

**Comment by ndunand** [comment]
Hi Gemma,

Obviously that depends on the hourly rate of the programmer, but in terms of time I would guess about 1-3 days depending on exact requirements.

**Comment by gemguardian** [comment]
Ok. As I understand of reading back and to your responds, this means I have to find a programmer that could do this? Or is it something you/your employer can do as long as someone funds it?

**Comment by ndunand** [comment]
Yes, your first statement is correct. I'm employed by a university and can't really allocate time for this in the foreseeable future.

However, were you to be able to (get someone to) program this feature, your contribution (i.e. the changes to the codebase itself) would be very welcome here, and I'd then make them available to all through a new release of the plugin.

**Comment by ndunand** [comment]
@gemguardian and news on this ?

**Comment by gemguardian** [comment]
My organization did not want to move forward on this. So sadly I have no resources to hire a programmer.

**Comment by jonathanjoegit** [comment]
Hello !
We could be interested too, let us know if somebody is doing something, we could test and help !
Have a nice day !
